packages/app/src/specs/SpecsList.cy.tsx

```diff
@@ -42,21 +41,24 @@ describe('<SpecsList />', { keystrokeDelay: 0 }, () => {
         acc.relative.length < spec.relative.length ? spec : acc
       , specs[0])
 
-      cy.get(inputSelector).type('garbage 🗑', { delay: 0 })
+      cy.findByLabelText(defaultMessages.specPage.searchPlaceholder)
```

Comment:
No need to answer or change anything here right now, but are we sticking w/ i18n in specs or not? there's a ton of discussion around this, it's not entirely clear what we landed on.

Reply:
Last I recall we were "sticking with it" as a practice but not to the point where we would hold up a PR that doesn't do it.